internal/server/v1/propertyvalues/core.go

```diff
@@ -107,11 +115,76 @@ func fetchSQL(
 	}
 	var matchColumn string
 	if direction == util.DirectionOut {
-		matchColumn = "subject_id"
+		matchColumn = subjectIdColumn
 	} else {
-		matchColumn = "object_id"
+		matchColumn = objectIdColumn
+	}
+
+	// Get triples for the specified nodes and properties.
+	triples, err := executeTriplesSQL(sqlClient, nodes, properties, matchColumn)
+	if err != nil {
+		return nil, err
+	}
+
+	// Collect all dcids from the triples and get their entity infos (name and type).
+	// NOTE: This will only fetch info on entities that are in the SQL database.
+	// If any dcids reference entities in base DC - those will not be fetched.
+	// If we want them as well, we'll need to make a remote mixer call to fetch them.
```

Comment:
I think this will be important to add, but at the same time i believe the logic from this PR should resolve UN's issue

Reply:
Maybe this can happen in bulk before the existing mixer merge stage?

Reply:
Ack. Yeah, doing it at that stage is possible but could get hairy.